cmd/apps/crossplane_app.go

```diff
@@ -0,0 +1,148 @@
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/alexellis/k3sup/pkg"
+	"github.com/alexellis/k3sup/pkg/config"
+	"github.com/alexellis/k3sup/pkg/env"
+	"github.com/alexellis/k3sup/pkg/helm"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallCrossplane() *cobra.Command {
+	var crossplane = &cobra.Command{
+		Use:          "crossplane",
+		Short:        "Install crossplane",
+		Long:         `Install crossplane`,
+		Example:      `  k3sup app install crossplane`,
+		SilenceUsage: true,
+	}
+
+	crossplane.Flags().StringP("namespace", "n", "crossplane-system", "The namespace used for installation")
+	crossplane.Flags().Bool("update-repo", true, "Update the helm repo")
+	crossplane.Flags().Bool("helm3", false, "Use helm3 instead of the default helm2")
+
+	crossplane.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath := getDefaultKubeconfig()
+
+		if command.Flags().Changed("kubeconfig") {
+			kubeConfigPath, _ = command.Flags().GetString("kubeconfig")
+		}
+
+		fmt.Printf("Using kubeconfig: %s\n", kubeConfigPath)
+
+		helm3, _ := command.Flags().GetBool("helm3")
+
+		if helm3 {
+			fmt.Println("Using helm3")
+		}
+		namespace, _ := command.Flags().GetString("namespace")
+
+		if namespace != "crossplane-system" {
+			return fmt.Errorf(`to override the namespace, install crossplane via helm manually`)
+		}
+
+		arch := getNodeArchitecture()
+		if !strings.Contains(arch, "64") {
```

Comment:
@alexellis this is currently my crude filtering of compatible architectures. Did you have something else in mind? We are hoping to support armhf *very* soon, at which point I would be happy to circle back around and remove this check :)

Reply:
I don't think it's enough, unless you support arm64?

Reply:
we do :+1: